### PR TITLE
Automate BZ 1321511 -- validate autocomplete content

### PR DESCRIPTION
**Pull request by svtkachenko**
https://bugzilla.redhat.com/show_bug.cgi?id=1321511
```python
% pytest -v tests/foreman/ui/test_hostgroup.py -k 'activation'
===================================================================== test session starts =====================================================================
platform linux2 -- Python 2.7.12, pytest-3.0.4, py-1.4.31, pluggy-0.4.0 -- /home/qui/code/venv/2/bin/python2
cachedir: .cache
rootdir: /home/qui/code/robottelo, inifile: 
plugins: xdist-1.15.0, html-1.12.0, cov-2.4.0
collected 8 items 

tests/foreman/ui/test_hostgroup.py::HostgroupTestCase::test_positive_check_activation_keys_autocomplete <- robottelo/decorators/__init__.py PASSED
tests/foreman/ui/test_hostgroup.py::HostgroupTestCase::test_positive_create_with_activation_keys <- robottelo/decorators/__init__.py PASSED

===================================================================== 6 tests deselected ======================================================================
=========================================================== 2 passed, 6 deselected in 85.65 seconds ===========================================================
```

### CODE REVIEW

File: tests/foreman/ui/test_hostgroup.py [hunk 1]
```diff
@@ -141,3 +153,73 @@ def test_positive_create_with_oscap_capsule(self):
...
+        4. Create an activation key pointed to view B in Library
+        5. Go to the new Hostgroup page, select content view B & Library,
+            click on the activation key tab and click in the input box
+
+        @Assert: Only the activation key for view B is listed
+
+        @BZ: 1321511
+        """
+        # Use setup entities as A and create another set for B.
+        cv = entities.ContentView(organization=self.organization).create()
```
Comment (oshtaier) [inline tests/foreman/ui/test_hostgroup.py]: is not so readable, pls use here name like cv_b and etc

File: tests/foreman/ui/test_hostgroup.py [hunk 2]
```diff
@@ -141,3 +153,73 @@ def test_positive_create_with_oscap_capsule(self):
...
+            self.hostgroup.assign_value(
+                locators['hostgroups.environment'], self.env.name)
+            self.hostgroup.assign_value(
+                locators['hostgroups.content_view'], cv.name)
+            # Switch to Activation Keys tab and click on input
+            # to load autocomplete
+            self.hostgroup.click(tab_locators['hostgroup.tab_activation_keys'])
+            self.hostgroup.click(locators['hostgroups.activation_keys'])
+            autocompletes = self.hostgroup.find_elements(
+                locators['hostgroups.autocomplete'])
```
Comment (oshtaier) [inline tests/foreman/ui/test_hostgroup.py]: it is autocomplete for activation keys, not hostgroups, so need to have proper name for locator

File: tests/foreman/ui/test_hostgroup.py [hunk 3]
```diff
@@ -141,3 +153,73 @@ def test_positive_create_with_oscap_capsule(self):
...
+        @id: 07906caf-871d-4d1f-8914-38027e71c16b
+
+        @Steps:
+
+        1. Create two content views A & B
+        2. Publish both content views
+        3. Create an activation key pointed to view A in Library
+        4. Create an activation key pointed to view B in Library
+        5. Go to the new Hostgroup page, select content view B & Library,
+            click on the activation key tab and click in the input box
```
Comment (oshtaier) [inline tests/foreman/ui/test_hostgroup.py]: do not see `and click in the input box` in your test. Is it autocomplete by click or values already present in the field? If already present then it is not autocomplete functionality, but just pre-loaded data

**Comment by oshtaier** [comment]
Pending comments

File: tests/foreman/ui/test_hostgroup.py (hunk 3 above)
Comment (svtkachenko) [inline tests/foreman/ui/test_hostgroup.py]: `self.hostgroup.click(locators['hostgroups.activation_keys'])`
`locators['hostgroups.activation_keys']` points to input field.

**Comment by oshtaier** [comment]
Not super happy about general logic, but ACK

**Pull request merged**